understanding/sign-language-prerecorded.html

```diff
@@ -58,118 +56,95 @@
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.2.6:</span>Sign Language (Prerecorded) (Level AAA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 1.2.6:</span>手話 (収録済) (レベル AAA)
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
                   <p>
-                     <a href="#dfn-sign-language-interpretation">Sign language interpretation</a> is provided for all <a href="#dfn-prerecorded">prerecorded</a>
-                     <a href="#dfn-audio">audio</a> content in <a href="#dfn-synchronized-media">synchronized media</a>.
+                     <a href="#dfn-synchronized-media">同期したメディア</a>に含まれているすべての<a href="#dfn-prerecorded">収録済</a>の<a href="#dfn-audio">音声</a>コンテンツに対して、<a href="#dfn-sign-language-interpretation">手話通訳</a>が提供されている。
+
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Videos can be accompanied by sign language.</dd>
+                  <dt>目標</dt>
+                  <dd>映像を手話が伴うようにすることができる。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Provide sign language interpretation for audio content in existing videos.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>既存の映像の音声コンテンツに手話通訳を提供する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People who are deaf or hard of hearing have more ways to understand multimedia content.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>ろう者又は難聴の人が、マルチメディアコンテンツを理解するためのより多くの手段を持つことができる。</dd> 
```

Comment:
「より多くの手段を持つ」は日本語としてこなれていない感じがします。「手段が増える」としてみてはどうでしょうか。

```suggestion
                  <dd>ろう者又は難聴の人にとって、マルチメディアコンテンツを理解するための手段が増える。</dd> 
```